File: BackEnd/eMenka/eMenka.Domain/Classes/CostAllocation.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace eMenka.Domain.Classes
{
    public class CostAllocation
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Abbreviation { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? EndDate { get; set; }
    }
}

File: eFenkaFrontend/src/app/models/cost-allocatoin/cost-allocation.ts
export class CostAllocation {
  id: number;
  name: string;
  abbreviation: string;
  startDate: Date;
  endDate: Date;

  constructor(id: number, name: string, abbreviation: string, startDate: Date, endDate: Date) {
    this.id = id;
    this.name = name;
    this.abbreviation = abbreviation;
    this.startDate = startDate;
    this.endDate = endDate;
  }
}
